course-v3/layouts/partials/head.html

```diff
@@ -0,0 +1,72 @@
+{{- $gtmId := getenv "GTM_ACCOUNT_ID" -}}
```

Comment:
Is this file just a copy of head.html from the base-theme? If yes, can we just remove it and hugo will pull it from there?